### PR TITLE
[SP-1990][BISERVER-10399] Using the Data Source Wizard in V5.0, when …

### PR DESCRIPTION
**Pull request by SergeyTravin**
…join fields are removed from the model, queries return no results.

### CODE REVIEW

**Comment by SergeyTravin** [comment]
@mdamour1976, @e-cuellar, @pentaho-nbaker, @kcruzada Could you review it please?

Root fix https://github.com/pentaho/data-access/pull/647

**Comment by SergeyTravin** [comment]
@mdamour1976, @e-cuellar, @pentaho-nbaker, @kcruzada Could you review it please?

The PR should be processed before end of August SP sprint ( 8/1/2015 )

**Comment by SergeyTravin** [comment]
@mdamour1976, @e-cuellar, @pentaho-nbaker, @kcruzada Could you review it please?

The PR should be processed before end of August SP sprint ( 8/1/2015 )
Root fix #647

File: src/org/pentaho/platform/dataaccess/datasource/wizard/EmbeddedWizard.java [hunk 1]
```diff
@@ -492,16 +499,38 @@ public void onDialogError( String errorMessage ) {
...
+    final EmbeddedWizard wizard = this;
+    //reload model for synchronize between GWT model and backend model http://jira.pentaho.com/browse/BISERVER-10399
+    GwtModelerServiceImpl ssd = new GwtModelerServiceImpl();
+    ssd.loadDomain( domainId, new  XulServiceCallback<Domain>() {
 
-    ModelerDialog.getInstance( this, new AsyncConstructorListener<ModelerDialog>() {
-      public void asyncConstructorDone( ModelerDialog dialog ) {
-        dialog.addDialogListener( listener );
+      @Override
+      public void error( String arg0, Throwable arg1 ) {
```
Comment (pentaho-nbaker) [inline src/org/pentaho/platform/dataaccess/datasource/wizard/EmbeddedWizard.java]: Wow these variable names are terrible

**Pull request merged**